packages/venia-concept/venia-upward.yml

```diff
@@ -21,6 +24,31 @@ response:
       use: appShell
   default: bundles
 
+imageOpt:
+  inline:
+    status: 301
+    body:
+      inline: ''
+    headers:
+      inline:
+        Location:
+          engine: mustache
+          provide:
+            width: $match.$1
+            path: $match.$2
+          template:
+            when:
+              - matches: env.USE_FASTLY
```

Comment:
This may not be the environment variable that ends up available, but it's easy to swap out.

Reply:
Is this documented anywhere?